README.md

```diff
@@ -1,11 +1,10 @@
 # Open Source Guides
-
 [![Build Status](https://github.com/github/opensource.guide/workflows/GitHub%20Actions%20CI/badge.svg)](https://github.com/github/opensource.guide/actions)
 
-Open Source Guides (https://opensource.guide/) are a collection of resources for individuals, communities, and companies who want to learn how to run and contribute to an open source project.
+Open Source Guides (https://opensource.guide/) is a collection of resources for individuals, communities, and companies who want to learn how to run and contribute to an open-source project.
```

Comment:
`are` is used because `Open Source Guides` is plural
```suggestion
Open Source Guides (https://opensource.guide/) are a collection of resources for individuals, communities, and companies who want to learn how to run and contribute to an open-source project.
```